app/assets/javascripts/scan_analytics.js

```diff
@@ -0,0 +1,210 @@
+        }]
+      };
+      chart1options = jQuery.extend(true, {}, options, chart1options);
+      var chart1 = new Highcharts.Chart(chart1options);
+
+     /*==========  Chart 2  defect density=====================*/
+      if (data['defect_density']){
+        var chart2options = {
+          chart: {
+            renderTo: 'chart2',
+            type: 'line',
+          },
+          title: {
+            text: 'Defect Density over period of time'
+          },
+          legend: {
+            align: 'right',
+            verticalAlign: 'top',
+            layout: 'vertical',
+            x: 0,
+            y: 100
+          },
+          yAxis: {
+            title: {
+              text: null
+            }
+          },
+          xAxis: {
+            categories: Object.entries(data['defect_density'][0].data).map(function(m){return m[0]})
+          },
+          series: [{
+            name: data['defect_density'] ? data['defect_density'][0].name : null,
+            data: data['defect_density'] ? Object.entries(data['defect_density'][0].data) : [],
+            color: '#7CB5EC'
+          }]
+        };
+        chart2options = jQuery.extend(true, {}, options, chart2options);
+        var chart2 = new Highcharts.Chart(chart2options);
+      }
+       /*==========  Chart 3 High Defects =====================*/
+      var chart3options = {
+        chart: {
+          renderTo: 'chart3',
+          type: 'bar',
+        },
+        title: {
+          text: 'High impact Outstanding Defect per Category'
+        },
+        legend: {
+          enabled: false
+        },
+        series: [{
+          color: '#800000',
+          name: 'Value',
+          data: Object.entries(data['high_impact_defects'])    
+        }],
+        xAxis: {
+          type: 'category',
+          title: {
+            text: 'Defect Category'
+          }
+        },
+        yAxis: {
+          title: {
+            text: 'Outstanding defects'
+          }
+        }
+      };
+      chart3options = jQuery.extend(true, {}, options, chart3options);
+      var chart3 = new Highcharts.Chart(chart3options);
+       /*==========  Chart 4 Medium Defects =====================*/
+      var chart4options = {
+        chart: {
+          renderTo: 'chart4',
+          type: 'bar',
+        },
+        title: {
+          text: 'Medium impact Outstanding Defect per Category'
+        },
+        legend: {
+          enabled: false
+        },
+        series: [{
+          name: 'Value',
+          data: Object.entries(data['medium_impact_defects']),
+          color: '#F7A35C'
+        }],
+        xAxis: {
+          type: 'category',
+          title: {
+            text: 'Defect Category'
+          }
+        },
+        yAxis: {
+          title: {
+            text: 'Outstanding defects'
+          }
+        },
+      };
+      chart4options = jQuery.extend(true, {}, options, chart4options);
+      var chart4 = new Highcharts.Chart(chart4options);
```

Comment:
Where is `chart4` used. If this variable is not used anywhere, remove it from here. Just call `new Highcharts`.